client/my-sites/plans-v2/constants.ts

```diff
@@ -164,3 +165,16 @@ export const SELECTOR_PLANS = [
 	PLAN_JETPACK_COMPLETE,
 	PLAN_JETPACK_COMPLETE_MONTHLY,
 ];
+
+/**
+ * Products included in each plan
+ */
+
+export const PLANS_INCLUDED_PRODUCTS: Record< string, string[] > = {
```

Comment:
The object is incomplete for now. I'd like to make sure this is the best way to check whether a product is included in a plan. I believe that until now, we've done this kind of checks indirectly based on the features of both plans and products, but this doesn't seem scalable given the new offer.

Reply:
If this is about legacy plans, then you can use this: https://github.com/automattic/wp-calypso/blob/master/client/lib/plans/index.js#L104-L104

Reply:
It's for new and legacy plans. I've just realized that product features (such as `FEATURE_JETPACK_BACKUP_DAILY`) are actually assigned the product name (`const FEATURE_JETPACK_BACKUP_DAILY = PRODUCT_JETPACK_BACKUP_DAILY;`), so yes, `planHasFeature` is the way to go 👍